packages/spor-react/src/theme/components/table.ts

```diff
@@ -1,161 +1,138 @@
-import { mode } from "@chakra-ui/theme-tools";
-import { baseBorder, baseText } from "../utils/base-utils";
-
-const helpers = createMultiStyleConfigHelpers(parts.keys);
+import { defineSlotRecipe } from "@chakra-ui/react";
+import { baseText } from "../utils/base-utils";
+import { useColorModeValue } from "../../color-mode";
 
 const numericStyles = {
   "&[data-is-numeric=true]": {
     textAlign: "end",
   },
 };
 
-const config = helpers.defineMultiStyleConfig({
-  baseStyle: (props) => ({
-    table: {
+export const tableSlotRecipe = defineSlotRecipe({
+  slots: [
+    "root",
+    "body",
+    "row",
+    "cell",
+    "columnHeader",
+    "caption",
+    "footer",
+    "header",
+  ],
+  className: "spor-table",
+  base: {
+    root: {
+      tableLayout: "fixed",
       borderCollapse: "collapse",
-      ...baseText("default", props),
       width: "100%",
+      ...baseText("default"),
     },
-    th: {
-      fontWeight: "bold",
-      textAlign: "start",
-    },
-    td: {
-      textAlign: "start",
+    body: {
+      ...baseText("default"),
     },
-    tfoot: {
-      tr: {
-        "&:last-of-type": {
-          th: { borderBottomWidth: 0 },
-        },
-      },
-    },
-    caption: {
-      marginTop: 4,
-      fontFamily: "heading",
-      textAlign: "center",
+    columnHeader: {
       fontWeight: "bold",
-      color: "currentColor",
+      textAlign: "start",
     },
-  }),
+  },
   variants: {
-    simple: (props) => ({
-      th: {
-        borderBottom: "sm",
-        borderColor: mode(`blackAlpha.200`, `whiteAlpha.300`)(props),
-        ...numericStyles,
-      },
-      td: {
-        borderBottom: "sm",
-        borderColor: mode(`blackAlpha.200`, `whiteAlpha.300`)(props),
-        ...numericStyles,
+    variant: {
+      line: {
+        cell: {
+          borderBottom: "sm",
+          borderColor: useColorModeValue(`blackAlpha.200`, `whiteAlpha.300`),
+          ...numericStyles,
+        },
+        row: {
+          borderBottom: "sm",
+          borderColor: useColorModeValue(`blackAlpha.200`, `whiteAlpha.300`),
+          ...numericStyles,
+          "&:last-of-type": {
+            td: {
+              borderBottomWidth: 0,
+            },
+          },
+        },
       },
-    }),
-    outline: (props) => ({
-      tbody: {
-        tr: {
+      outline: {
+        cell: {
+          border: "sm",
+          borderColor: "blackAlpha.200",
+          ...numericStyles,
+          _first: {
+            borderLeft: "none",
+          },
+        },
+        row: {
           transitionDuration: "fast",
           transitionProperty: "background-color, box-shadow",
           _hover: {
-            ...baseBorder("hover", props),
-            outlineOffset: "-2px",
-            backgroundColor: mode(
-              props.colorScheme === "grey" ? "mint" : "coralGreen",
-              "whiteAlpha.200",
-            )(props),
+            backgroundColor: "blackAlpha.200",
```

Comment:
Hvorfor byttes denne til black i steden for white?

Reply:
godt oppdaget, det er nok en feil